client/src/pages/chatlink/captcha.js

```diff
@@ -1,10 +1,11 @@
 import loadJS from '../../utils/loadJS';
+import captcha_config from './captcha.json'
```

Comment:
Why can't we keep it in env ?

Reply:
The ENV variables become unavailable to the static files generated after build. Having it in a configuration file will make the value available even after the build.

Reply:
We don't need env after static files are generated. Webpack replaces while building. We can't have it as hard-coded because it's different for different developers. Can't commit the file to git.

Reply:
Fixed the issue in latest commit, please check out